packages/webrtc-core/src/index.ts

```diff
@@ -5,8 +5,11 @@ import * as media from './media';
 export * from './device/device-management';
 export * from './media/local-camera-track';
 export * from './media/local-display-track';
+export * from './media/local-computer-audio-track';
 export * from './media/local-microphone-track';
 export * from './media/local-track';
+export * from './media/local-video-track';
+export * from './media/local-audio-track';
 export * from './peer-connection';
 export * from './peer-connection-utils';
 export { media };
```

Comment:
I think WebrtcError needs to be exported here as well (I think this was WcmeError before, and that was being exported in the original code).

Reply:
done